packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -528,7 +531,7 @@ class CloudFormation {
           log(e);
           if (e.code === 'Throttling' && e.retryable) {
             setTimeout(() => {
-              resolve(this.describeStack(cfnNestedStackParams, maxTry - 1, timeout));
+              resolve(this.describeStack(cfnNestedStackParams, maxTry - 1, timeout + CFN_POLL_TIME_BACKOFF));
```

Comment:
We can use same logic of exponential backoff here

```suggestion
              resolve(this.describeStack(cfnNestedStackParams, maxTry - 1, 2^ (this.describeEventsRetries) * timeout));
              this.describeEventsRetries++;
```